store/config.go

```diff
@@ -114,40 +131,40 @@ func (c Config) BridgeResponseURL() *url.URL {
 
 // ChainID represents the chain ID to use for transactions.
 func (c Config) ChainID() uint64 {
-	return uint64(c.viper.GetInt64("ChainID"))
+	return uint64(c.viper.GetInt64(c.envVarName("ChainID")))
```

Comment:
A stretch goal that would be great would be to double down on reflection and see if we could pass the calling method's name to `c.envVarName(...)`, or do meta the go way™ and code generate into a separate file.

Reply:
I did look into this, it's entirely possible - the runtime keeps the function call in the stack and you can inspect it. But thought it a bit too clever.